docs/2_2_common_mechanisms.adoc

```diff
@@ -672,14 +682,18 @@ t_i &:= t_{i-1} + T_{\mathit{interval, i}} \qquad i = 1,2,3,{...}
 |===
 
 |latexmath:[T_{\mathit{interval, i}} > 0]
-|The time interval from the previous Clock tick to the current Clock tick, specified differently for the different Clock types.
+|Generally, the time interval from the previous Clock tick to the current Clock tick.
+ An exception applies to a tunable Clock whose interval has just changed in <<EventMode>> or <<ClockActivationMode>>.
+ In this specific case, the new interval is relative to the time of the current <<EventMode>> or <<ClockActivationMode>>, not to the time of the previous Clock tick.
+ The interval is provided differently for the different Clock types.
+
 |===
 
 Constant periodic Clock::
 is a time-based periodic clock that defines its <<intervalDecimal>> and optionally <<intervalCounter>>, <<shiftCounter>> and <<shiftDecimal>> in the <<modelDescription.xml>>.
 
-Fixed periodic Clock::
-is a time-based periodic Clock which ticks with an arbitrary, but fixed interval latexmath:[T_{\mathit{interval}}] starting after an arbitrary latexmath:[T_{\mathit{shift}}], both interval and shift can be changed by the importer.
```

Comment:
why drop this part of the sentence?
I think we added this to make sure implementers know that the importer can adjust the actual intervals according to its own needs, like a specific, fixed, step size.